### DISCUSSION TITLE
Mouse ?

### DISCUSSION THREAD

**Discussion by Al-fata-tayeb** [ikemen-engine/Ikemen-GO#1305] [Feature Request]
.

**Comment by jenngra505** [comment]
Interesting concept but the phrasing could use some work so let me write my case for why mouse inputs should be used in IKEMEN GO.
In modern fighting games, notably Street Fighter 6, you can navigate menus and even character-select screens using your mouse. But it would be interesting to have mouse support beyond menus for several reasons. First, as an alternative to keyboard inputs for masochists with a video idea like this. https://www.youtube.com/watch?v=EQfFmylgTKI More importantly though, it would be useful to have some SCtrls to react to mouse clicks regardless of control layout for creating gimmick characters in the vein of WINMUGEN_Paint or Cyber-x64's Cookie Clicker. This could make running DOOM in IKEMEN a tad easier /hj.
